### ISSUE TITLE
[Standardization] KaaS Cluster Management Gitops Controller

### ISSUE THREAD

**Issue by cah-hbaum** [SovereignCloudStack/issues#419]
This issue was created to provide a discussion ground for possible future standards. It is derived from #181 and on of the points not assigned any issue yet.
This issue should discuss the introduction of a Gitops Controller for Cluster Management.

---

## Definition of Done:

- [x] [Cluster stacks](https://github.com/SovereignCloudStack/cluster-stack-operator) are available, which unblocks this issue
- [ ] Conclusions are reached about all discussion topics
- [ ] A new issue for the creation of a standard (proposal) is created and linked

**Comment by cah-hbaum** [comment]
Notes from prioritization meeting 06-10-23:

* push the cluster control yaml to a GIT
* an event/controller checks the files and updates the cluster
-> should be delayed by 3+ months until the cluster stacks are available
